opencti-platform/opencti-front/src/private/components/settings/sub_types/scaleConfiguration/ScaleConfiguration.tsx

```diff
@@ -188,14 +189,47 @@ const ScaleConfiguration: FunctionComponent<EntitySettingScaleProps> = ({
     });
   };
 
+  const currentScaleName = findSelectedScaleName(tickDefinition);
```

Comment:
When I customize an existing scale, I loose the selected scale template
[screen-capture (7).webm](https://github.com/OpenCTI-Platform/opencti/assets/53513584/579a5713-b8bc-4913-a359-bfa2773115a8)

Reply:
Yes, when you start to customize, it's not a template anymore, and I didn't find a way to display "custom", it just displays nothing in the selector. But when you open it again after saving, it displays custom, and if you change and select another template, you'll be able to select again your custom config. @RomuDeuxfois

Reply:
Yes, every other use case work very well !